cli/data/playbooks/deploy/deploy.yaml

```diff
@@ -41,6 +41,17 @@
     - name: Deploy debug_tools
       ansible.builtin.include_role:
         name: debug_tools
+    - name: Install podman
+      ansible.builtin.package:
+        name:
+          - podman
+    - name: Install other dependencies
+      ansible.builtin.package:
+        name:
+          - bash-completion
+          - nmap
+          - python3-libsemanage
+          - python3-psycopg2
```

Comment:
This is already listed in `install_requires` so we can drop this
```suggestion
```

However, now I need to be kept honest. https://galaxy.ansible.com/ui/repo/published/community/postgresql/docs/ lists external requirements but it doesn't list where. Is it the control node or the node it's really running on? That may be important for external database support.

Reply:
Exactly, otherwise you get https://github.com/theforeman/foreman-quadlet/actions/runs/14798416218/job/41551130603#step:10:251

Reply:
> On the controlled node (so the external db)

So the installation really needs to be in the PostgreSQL role? And we can drop it from `setup.py`?

Then we'd need to do the same in the place where we use the crypto role.

Reply:
We can add them as extras in setup.py. In the final system we can then add a Suggests on the extra. By default dnf will install that, reducing the transactions. I'd hope that ansible ensuring an already installed package is cheap